src/site/content/en/mini-apps/mini-app-open-source-projects/index.md

```diff
@@ -0,0 +1,295 @@
+  </figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="preact-web.png" alt="The Preact kbone template demo app opened in the web browser. Inspecting the DOM structure shows the to-be-expected markup based on the Preact component code.">
+  <figcaption class="w-figure">
+   The Preact kbone template demo app opened in the web browser. Note the DOM structure.
+  </figcaption>
+</figure>
+
+## kbone-ui
+
+The [kbone-ui](https://wechat-miniprogram.github.io/kbone/docs/ui/intro/) project
+([open source on GitHub](https://github.com/wechat-miniprogram/kbone-ui)) is a UI framework that
+facilitates both mini app development as well as Vue.js development with kbone. The kbone-ui
+components emulate the look and feel of
+[WeChat's "native" mini app components](https://developers.weixin.qq.com/miniprogram/dev/component/)
+(also see [Components](/mini-app-components/) above). A
+[demo](https://wechat-miniprogram.github.io/kboneui/ui/#/) that runs directly in the browser lets
+you explore the available components.
+
+<figure class="w-figure">
+  <img src="kbone-ui.png" alt="Demo of the kbone-ui framework showing form-related components like radio buttons, switches, inputs, and labels." width="320">
+  <figcaption class="w-figure">
+   The kbone-ui demo showing form-related components.
+  </figcaption>
+</figure>
+
+## WeUI
+
+[WeUI](https://github.com/Tencent/weui) is a set of basic style libraries consistent with WeChat's
+native visual experience. The official WeChat design team has tailored designs for WeChat internal
+web pages and WeChat mini apps to make users' perception of use more uniform. It includes components
+such as `button`, `cell`, `dialog`, `progress`, `toast`, `article`, `actionsheet`, and `icon`. There
+are different versions of WeUI available like [weui-wxss](https://github.com/Tencent/weui-wxss/) for
+WeChat mini apps styled with WXSS (see [Styling](/mini-app-markup-styling-and-scripting/#styling) above),
+[weui.js](https://github.com/weui/weui.js/) for web apps, and
+[react-weui](https://github.com/weui/react-weui/) for WeChat React components.
+
+<figure class="w-figure">
+  <img src="weui.png" alt="Demo of the WeUI framework showing form-related components, namely switches." width="450">
+  <figcaption class="w-figure">
+   The WeUI demo app showing switches.
+  </figcaption>
+</figure>
+
+## Omi
+
+[Omi](https://tencent.github.io/omi/) is a self-proclaimed front-end cross-frameworks framework
+([open source on GitHub](https://github.com/Tencent/omi). It merges Web Components, JSX, Virtual
+DOM, functional style, observer or Proxy into one framework with tiny size and high performance. Its
+aim is to let developers write components once and use them everywhere, such as Omi, React, Preact,
+Vue.js, or Angular. Writing Omi components is very intuitive and free of almost all boilerplate.
```

Comment:
```suggestion
Vue.js, or Angular. I personally found writing Omi components is very intuitive and
free of almost all boilerplate.
```
Because this is subjective I would like to emphasize that this is your personal opinion

Reply:
Good point!